e2e/components/tabs/tabs.e2e.ts

```diff
@@ -0,0 +1,24 @@
+describe('tabs', function () {
+  describe('basic behavior', function () {
+    beforeEach(function() {
+      browser.get('/tabs');
+    });
+    it('should change tabs when the label is clicked', function () {
+      element(by.css('.md-tab-label:nth-of-type(2)')).click();
+      expect(element(by.css('.md-tab-label:nth-of-type(2)')).getAttribute('class'))
+          .toMatch('md-active');
+      expect(element(by.css('.md-tab-body:nth-of-type(2)')).getAttribute('class'))
+          .toMatch('md-active');
+
+      element(by.css('.md-tab-label:nth-of-type(1)')).click();
+      expect(element(by.css('.md-tab-label:nth-of-type(2)')).getAttribute('class')).not
```

Comment:
To assert that the tabs contain a class you can do something like this:

```js
// Note: $$('.abc') === element.all(by.css('.abc'))
let tabsActive = $$('.md-tab-label').map((item, index) => {
  // Note: I am creating an array that contains {index, isActive} where isActive means that
  // it has the active class
  return {
    index: index,
    isActive: item.getAttribute('class').then(
        classAttr => !!classAttr.match('dropdown'))
  };
});

expect(tabsActive).toEqual([
  {index: 0, isActive: false}, 
  {index: 1, isActive: true},
  {index: 2, isActive: false}
]);
```

Or you can just do:

```js
let tabsActive = $$('.md-tab-label')
    .map(item => item.getAttribute('class')
        .then(classAttr => !!classAttr.match('dropdown')));

expect(tabsActive).toEqual([false, true, false]);
```